.github/workflows/docs-update.yml

```diff
@@ -0,0 +1,70 @@
+name: Update Docs
+
+on:
+  push:
+    tags:
+      # https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string -- github doesn't like this regex??
```

Comment:
[Question] What about not liking it?

Reply:
Not sure, it just doesn't work.
Splitting it into two separate regexps works though.